utility/src/main/java/org/oppia/android/util/parser/math/MathBitmapModelLoader.kt

```diff
@@ -225,6 +225,9 @@ class MathBitmapModelLoader private constructor(
       }
 
       override fun drawPath(path: Path, paint: Paint) {
+        // TODO(#3616): Migrate to the proper SDK 35+ APIs.
```

Comment:
Is this the correct issue? It doesn't seem to correspond to the specific API that's now deprecated here.